bin/esi-install-image

```diff
@@ -129,15 +129,16 @@ class ServiceImageManager(object):
         for package in matches:
             return package['version']
 
-    def install(self, tarball):
+    def install(self, tarball, version=None):
         ### Decompress image
-        decompress_cmd = "/bin/tar xJf {0}".format(tarball)
+        decompress_cmd = "/bin/tar vxJf {0}".format(tarball)
```

Comment:
I removed the v option because all it caused was excess noise.  Is it needed for something?